hail/src/test/scala/is/hail/io/compress/BGzipCodecSuite.scala

```diff
@@ -114,4 +115,57 @@ class BGzipCodecSuite extends SparkSuite {
     }
     p.check()
   }
+
+  @Test def testVirtualSeek() {
+    sc.hadoopConfiguration.setLong("mapreduce.input.fileinputformat.split.minsize", 1L)
```

Comment:
Modifying global state in the tests is dangerous.  Things like this should clean up in general.  However, you don't appear to be using a Hadoop input format here, so I don't think this is doing anything.

Reply:
I copied this from the above test. Is it also unnecessary there?

Reply:
No, it is necessary there because it calls `sc.textFile` which uses a Hadoop input format.  But it should clean up after itself.  I'll take care of that.